node/node.d.ts

```diff
@@ -170,6 +170,7 @@ declare module NodeJS {
         removeAllListeners(event?: string): EventEmitter;
         setMaxListeners(n: number): void;
         listeners(event: string): Function[];
+        _events?:string[];
```

Comment:
Use `['_events']` if you want to. This makes it explicit that you are using undocumented code paths